File: 3110-score-of-a-string/3110-score-of-a-string.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        diff = 0
        n = len(s)
        for i in range(n - 1):
            diff += abs((ord(s[i]) - ord('a')) - (ord(s[i + 1]) - ord('a')))
        return diff

File: LeetCode/3110-score-of-a-string/3110-score-of-a-string.cpp
class Solution {
public:
    int scoreOfString(string s) {
        int diff = 0;
        int n = s.length();
        for (int i = 0; i < n - 1; i++) {
            diff += abs((s[i] - 'a') - (s[i+1] - 'a'));
        }
        return diff;
    }
};